pkg/mirrorregistries/generator.go

```diff
@@ -96,3 +102,30 @@ func ExtractLocationMirrorDataFromRegistriesFromToml(registriesConfToml string)
 
 	return registriesConfList, nil
 }
+
+// GenerateInsecurePolicyJSON returns a base64 encoded minimal policy.json that disables signature enforcement
+func (m *mirrorRegistriesConfigBuilder) GenerateInsecurePolicyJSON() (string, error) {
+	if !m.ForceInsecurePolicy {
+		return "", nil
+	}
+
+	policy := map[string]interface{}{
+		"default": []map[string]string{
+			{"type": "insecureAcceptAnything"},
+		},
+		"transports": map[string]interface{}{
```

Comment:
Recently saw a customer that had "docker" that needed to be overwritten too https://redhat-internal.slack.com/archives/CUPJTHQ5P/p1764969432607049?thread_ts=1764952151.009109&cid=CUPJTHQ5P